.github/workflows/long-running-azure.yaml

```diff
@@ -250,6 +250,11 @@ jobs:
           client-id: ${{ secrets.AZURE_SP_TESTS_APPID }}
           tenant-id: ${{ secrets.AZURE_SP_TESTS_TENANTID }}
           subscription-id: ${{ secrets.AZURE_SUBSCRIPTIONID_TESTS }}
+      - name: Azure get token
+        uses: azure/cli@v2
+        with:
+          inlineScript: |
+            az account get-access-token --scope https://containerregistry.azure.net/.default --output none
```

Comment:
Thanks for finding this fix! 🚀 

Only two requests: 
- please add a description to the PR that explains why this fixes the issue.
- Please add a comment here in the code that also explains why this is done. and why that particular scope is being requested, and where is the token being set and retrieved. In other words, how and why does this fix the issue? Also a link to the issue description would help.